support/iconsClassesGenerator.js

```diff
@@ -0,0 +1,154 @@
+			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(
+				svgData,
+				focusColor
+			)}") !important;
+		}`;
+
+	return `${activeIconCSS} ${defaultIconCSS} ${disableIconCSS} ${hoverIconCSS} ${focusIconCSS}`;
+};
+
+let iconsCSSContent = '';
+
+for (const [cKEditorIcon, clayIcon] of Object.entries(iconsConfig.icons)) {
+	let directionClass = '';
+	let svgDataList = [];
+
+	if (typeof clayIcon === 'string') {
+		const svgData = fs.readFileSync(
+			`${sourceIconsPath}/${clayIcon}.svg`,
+			'utf8'
+		);
+
+		iconsCSSContent += getCSS(svgData, cKEditorIcon);
+		//svgDataList = [fs.readFileSync(`${sourceIconsPath}/${clayIcon}.svg`, 'utf8')];
+	} else {
+		if (clayIcon.ltr) {
+			const svgData = fs.readFileSync(
+				`${sourceIconsPath}/${clayIcon.ltr}.svg`,
+				'utf8'
+			);
+
+			iconsCSSContent += getCSS(svgData, cKEditorIcon, 'ltr');
+		}
+
+		if (clayIcon.rtl) {
+			const svgData = fs.readFileSync(
+				`${sourceIconsPath}/${clayIcon.rtl}.svg`,
+				'utf8'
+			);
+
+			iconsCSSContent += getCSS(svgData, cKEditorIcon, 'rtl');
+		}
+
+		//directionClass = `.cke_${clayIcon.direction}`;
+		//svgData = fs.readFileSync(`${sourceIconsPath}/${clayIcon.icon}.svg`, 'utf8');
+	}
+
+	/*
+	const activeIconCSS = 
+		`${directionClass}.cke_hidpi .cke_button.cke_button_on .cke_button__${cKEditorIcon}_icon,
+		${directionClass} .cke_button.cke_button_on .cke_button__${cKEditorIcon}_icon {
+			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(svgData, activeColor)}") !important;
+		}`;
+
+	const defaultIconCSS = 
+		`${directionClass}.cke_hidpi .cke_button .cke_button__${cKEditorIcon}_icon,
+		${directionClass} .cke_button .cke_button__${cKEditorIcon}_icon {
+			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(svgData, defaultColor)}") !important;
+		}`;
+
+	const disableIconCSS = 
+		`${directionClass}.cke_hidpi .cke_button.cke_button_disabled .cke_button__${cKEditorIcon}_icon,
+		${directionClass} .cke_button.cke_button_disabled .cke_button__${cKEditorIcon}_icon {
+			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(svgData, disableColor)}") !important;
+		}`;
+
+	const hoverIconCSS = 
+		`${directionClass}.cke_hidpi .cke_button:not(.cke_button_disabled):hover .cke_button__${cKEditorIcon}_icon,
+		${directionClass} .cke_button:not(.cke_button_disabled):hover .cke_button__${cKEditorIcon}_icon {
+			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(svgData, hoverColor)}") !important;
+		}`;
+
+	const focusIconCSS = 
+		`${directionClass}.cke_hidpi .cke_button:not(.cke_button_disabled):focus .cke_button__${cKEditorIcon}_icon,
+		${directionClass} .cke_button:not(.cke_button_disabled):focus .cke_button__${cKEditorIcon}_icon {
+			background: url("data:image/svg+xml;charset=utf8,${encodeSvgData(svgData, focusColor)}") !important;
+		}`;
+
+		iconsCSSContent += `${activeIconCSS} ${defaultIconCSS} ${disableIconCSS} ${hoverIconCSS} ${focusIconCSS}`;
+	*/
```

Comment:
Are we still saving all these comments? :D 

Not sure if this is something you want to keep... but if you do, one option is to create a bogus commit adding it and one removing it, so you will have it in the git history :D

Reply:
Ups, no, I have it already in the history, just forgot to remove it, thx